src/javascript/DesignSystem/SiteSwitcherDisplay/SiteSwitcher.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {Menu, MenuItem, withStyles} from '@material-ui/core';
+import {Button, Typography} from '@jahia/design-system-kit';
+import {ChevronDown} from 'mdi-material-ui';
+
+const styles = theme => ({
+    siteSwitcher: {
+        marginRight: `${theme.spacing.unit}!important`
+    }
+});
+
+const SiteSwitcher = ({siteKey, siteNodes, onSelectSite, classes}) => {
+    const [anchorEl, setAnchorEl] = useState(null);
+
+    const handleClick = event => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    const siteNode = siteNodes.find(siteNode => siteNode.name === siteKey);
+
+    return (
+        <>
+            <Button aria-owns={anchorEl ? 'site-switcher' : null}
+                    size="compact"
+                    color="inverted"
+                    aria-haspopup="true"
+                    data-cm-role="site-switcher"
+                    className={classes.siteSwitcher}
+                    onClick={handleClick}
+            >
+                <Typography noWrap variant="zeta" color="inherit">
+                    {siteNode.displayName}
+                </Typography>
+                &nbsp;
+                <ChevronDown fontSize="small" color="inherit"/>
+            </Button>
+            <Menu id="site-switcher" anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={handleClose}>
```

Comment:
Id for an element of a design system is dangerous as id must be unique for the whole page.

Reply:
what you suggest? should I remove it?